src/app/destinyTrackerApi/d2-bulkFetcher.ts

```diff
@@ -1,7 +1,17 @@
-import _ from 'underscore';
+import * as _ from 'underscore';
 import { D2ItemListBuilder } from './d2-itemListBuilder';
+import { DimStore } from '../inventory/store/d2-store-factory.service';
+import { DtrBulkItem } from '../item-review/destiny-tracker.service';
+import { D2ReviewDataCache } from './d2-reviewDataCache';
+import { IPromise, IQService, IHttpService } from 'angular';
 
 class D2BulkFetcher {
+  _reviewDataCache: D2ReviewDataCache;
+  _loadingTracker: any;
+  _trackerErrorHandler: any;
+  _itemListBuilder: D2ItemListBuilder;
+  $http: IHttpService;
+  $q: IQService;
```

Comment:
Doesn't need to be part of this, but in general we're moving towards just regular modules and classes rather than Angular DI.